sign/dilithium/mode1.go

```diff
@@ -0,0 +1,105 @@
+// WARNING This file is auto-generated from
```

Comment:
Recall to regenerate the files. These ones still have the previous comment.

Reply:
You were looking at an outdated version of the file:

<img width="669" alt="Screenshot 2020-05-11 at 12 42 57" src="https://user-images.githubusercontent.com/9975/81558079-f5fa2900-9384-11ea-99e3-a6311496ae29.png">